### PR TITLE
[NUI] Add GrabHandleColor property

### PR DESCRIPTION
**Pull request by wonrst**
### Description of Change ###

GrabHandleColor property can set the color of grab handle in TextField/TextEditor.

exmaple)
field.GrabHandleColor = new Vector4(0.0f, 1.0f, 0.0f, 1.0f);

this patch should be merged after below patches are applied.
https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-toolkit/+/255726/
https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-csharp-binder/+/255739/

### CODE REVIEW

File: src/Tizen.NUI/src/public/BaseComponents/TextEditor.cs [hunk 1]
```diff
@@ -1137,6 +1137,27 @@ public int PrimaryCursorPosition
...
         }
 
+        /// <summary>
+        /// The GrabHandleColor property.
+        /// </summary>
+        /// <remarks>
+        /// The property cascade chaining set is possible. For example, this (textEditor.GrabHandleColor.X = 0.1f;) is possible.
+        /// </remarks>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Vector4 GrabHandleColor
```
Comment (rabbitfor) [inline src/Tizen.NUI/src/public/BaseComponents/TextEditor.cs:1147]: Please use `Color` type instead `Vector4` here.

File: src/Tizen.NUI/src/public/BaseComponents/TextField.cs [hunk 2]
```diff
@@ -1251,6 +1251,27 @@ public int PrimaryCursorPosition
...
         }
 
+        /// <summary>
+        /// The GrabHandleColor property.
+        /// </summary>
+        /// <remarks>
+        /// The property cascade chaining set is possible. For example, this (textField.GrabHandleColor.X = 0.1f;) is possible.
+        /// </remarks>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Vector4 GrabHandleColor
```
Comment (rabbitfor) [inline src/Tizen.NUI/src/public/BaseComponents/TextField.cs:1261]: Ditto.

**Comment by rabbitfor** [comment]
Please note that, if you want `GrabHandleColor` to be used in a theme styling, you need to add same property to the corresponding style classes, such as `TextFieldStyle`.

**Comment by wonrst** [comment]
I will add same property to TextFieldStyle and TextEditorStyle(after implement) soon, thanks.

File: src/Tizen.NUI/src/public/BaseComponents/TextEditorBindableProperty.cs [hunk 3]
```diff
@@ -955,5 +955,23 @@ public partial class TextEditor
...
+            var textEditor = (TextEditor)bindable;
+            if (newValue != null)
+            {
+                Tizen.NUI.Object.SetProperty((System.Runtime.InteropServices.HandleRef)textEditor.SwigCPtr, TextEditor.Property.GrabHandleColor, new Tizen.NUI.PropertyValue((Color)newValue));
+            }
+        }),
+        defaultValueCreator: (BindableProperty.CreateDefaultValueDelegate)((bindable) =>
+        {
+            var textEditor = (TextEditor)bindable;
+            Vector4 temp = new Vector4(0.0f, 0.0f, 0.0f, 0.0f);
```
Comment (rabbitfor) [inline src/Tizen.NUI/src/public/BaseComponents/TextEditorBindableProperty.cs:972]: Please also change this to Color. Current code will cause unnecessary type conversion.

File: src/Tizen.NUI/src/public/BaseComponents/TextFieldBindableProperty.cs [hunk 4]
```diff
@@ -1006,6 +1006,23 @@ public partial class TextField
...
+            var textField = (TextField)bindable;
+            if (newValue != null)
+            {
+                Tizen.NUI.Object.SetProperty((System.Runtime.InteropServices.HandleRef)textField.SwigCPtr, TextField.Property.GrabHandleColor, new Tizen.NUI.PropertyValue((Color)newValue));
+            }
+        }),
+        defaultValueCreator: (BindableProperty.CreateDefaultValueDelegate)((bindable) =>
+        {
+            var textField = (TextField)bindable;
+            Vector4 temp = new Vector4(0.0f, 0.0f, 0.0f, 0.0f);
```
Comment (rabbitfor) [inline src/Tizen.NUI/src/public/BaseComponents/TextFieldBindableProperty.cs:1022]: Ditto.

**Review by rabbitfor — APPROVED** [review]
LGTM. Please make sure that the related dali patches are merged before merging it.

**Comment by wonrst** [comment]
I rebase the 4 commits and merge them into one. There is no change in the code.

**Pull request merged**